cmd/init.go

```diff
@@ -258,6 +256,10 @@ func install(config *initCommandConfig) error {
 		return errors.Errorf("%v", perr)
 
 	}
+	err := setProjectName(config.RootCommandConfig, config.projectName)
```

Comment:
I would suggest you avoid passing in the rootConfig here. The only reason why you are doing it is because `setProjectName` then calls `dir, perr := getProjectDir(config)`. I would suggest doing that call here, and passing in the projectDir as a string to setProjectName. I want to avoid the proliferation of functions that require the entire configuration struct to be passed in even though it's needed just to get something else.

Reply:
Done